ui/src/components/Table/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from "./TableComponents";
```

Comment:
This is a bit different from other /components. Each index.tsx file of a component in @flow/components is a single component, but here you have 2 components under Table/ being exported.
Please move TableComponents to @flow/components/Table as the index, and then create a DataTable either in components/ (if its generic enough) or in to @flow/features/BottomPanel/components/DataTable directly

Reply:
Updated to create the DataTable as a separate component